### ISSUE TITLE
Working with IP150? How reliable is this plugin?

### ISSUE THREAD

**Issue by morphix9** [e-mcgee/paradox_platform#35]
Hey guys,

I'm looking for a solution to get Paradox into HomeKit. I have the IP150 module and I'm running homebridge.
Will it work with IP150? How reliable is it, considering that the last update of this plugin was 2 years ago (no offence to the author, but I'm really curious).

**Comment by sebklaus** [comment]
@morphix9 I've been using this for over 3 years and it's been working really well.
I, for some reason, cannot make version 0.0.380 work, so I'm still on 0.0.350, but that's the only "issue" I have come across.

If I have time, I might look into getting https://github.com/ParadoxAlarmInterface/pai up and running, but there is no technical reason to switch. This plugin works just fine in my experience.

@e-mcgee Thank you for putting it together!

**Comment by morphix9** [comment]
> @morphix9 I've been using this for over 3 years and it's been working really well.
> I, for some reason, cannot make version 0.0.380 work, so I'm still on 0.0.350, but that's the only "issue" I have come across.
> 
> If I have time, I might look into getting https://github.com/ParadoxAlarmInterface/pai up and running, but there is no technical reason to switch. This plugin works just fine in my experience.
> 
> @e-mcgee Thank you for putting it together!

Great to hear. Do you connect over IP150? Do you have multiple partitions or just one?
Guess, I need to give it a try then :)

**Comment by sebklaus** [comment]
@morphix9 Yes, IP150 and one partition, but multiple partitions work (I had it set up for a bit and moved some sensors in, that I temporarily wanted to isolate).

It's definitely worth giving a go. I've been very happy with it.

**Comment by danielmekis** [comment]
> @morphix9 Yes, IP150 and one partition, but multiple partitions work (I had it set up for a bit and moved some sensors in, that I temporarily wanted to isolate).
> 
> It's definitely worth giving a go. I've been very happy with it.

@sebklaus Is there any way to ask help to make it work? I tried several times without any luck. I also have IP150 and have no idea how to make it work

**Comment by vsidlu** [comment]
@sebklaus did you manage to install plugin? 
after that you need to configure config.json - look at sample - [https://github.com/e-mcgee/paradox_platform/blob/master/config.json.sample.json](url)

**Comment by sebklaus** [comment]
@danielmekis 

I'll try, sure.

This is my package.json for this plugin:

```
{
  "dependencies": {
    "homebridge-paradox-securitysystem": "0.0.350",
  }
}
```

As I mentioned before, I'm still on version 0.0.350 as the latest doesn't work for me.

This is part of my config.json with an example for each type of sensor (I use). I'm not including anything else as I know the below works.

```
{
    "platform": "Paradox",
    "ip": "[INSERT IP OF IP150]",
    "port": 10000,
    "password": "[INSERT PASSWORD FOR IP150]",

    COMMENT: I don't use mqtt, so it's disabled.
    
    "mqtt": false,
    "mqtturl": "mqtt://[INSERT ANY IP]",
    "qos": 0,
    "retain": false,
    "mqttusername": "",
    "mqttpassword": "",
    "logindelay": 3800,
    "polldelay": 5000,
    "waitafterlogin": 600,
    "delaybetweencmds": 250,
    "logoutdelay": 500,
    "logindelayafterreconnect": 1000,
    "accessories": [
        {
            "name": "Area 1",
            "config": {
                "type": "Alarm",
                "partition": "0"
            }
        },
        {
            COMMENT: I only have one partition. Setting the name to "inactive" stops it from showing up in Home app (as per documentation).

            "name": "inactive",
            "config": {
                "type": "Alarm",
                "partition": "1"
            }
        },
        {
            "name": "Connected",
            "config": {
                "type": "Connected"
            }
        },
        {
            "name": "Motion sensor",
            "config": {
                "type": "Motion Sensor",
                "zone": [INSERT SENSOR NUMBER/ ZONE],
                "debounceDelay": 0
            }
        },
        {
            "name": "Contact sensor",
            "config": {
                "type": "Contact Sensor",
                "zone": [INSERT SENSOR NUMBER/ ZONE],
                "debounceDelay": 0
            }
        },
        {
            COMMENT: I have a PGM hooked to my garage gate motor

            "name": "PGM",
            "config": {
                "type": "Garage Door",

                COMMENT: This is the number/ zone of the contact sensor attached to my garage gate. This way it correctly registers the state of the gate. Otherwise it only works off the open/ close time.

                "zone": [INSERT SENSOR NUMBER/ ZONE],
                "pgm": [INSERT PGM NUMBER],

                COMMENT: It takes my garage gate roughly 15 seconds to open/ close. Set the correct time below.

                "doorOpensInSeconds": 15,
                "debounceDelay": 0
            }
        }
    ]
}
```

I hope the above, in combination with the official documentation, will be helpful for you/ others that stumble across this and want to have a play.

**Comment by vsidlu** [comment]
what version of homebridge do you have? 
does it work on homebridge v1.3.0?

**Comment by sebklaus** [comment]
I'm running homebridge 1.3.0, yes, but plugin version 0.0.380 never worked for me. It's nearly 2 years old and I've been using this plugin for 3+ years. So I'm happily sticking with 0.0.350.

**Comment by vsidlu** [comment]
do you see errors like this arming - 

[2/22/2021, 2:35:05 PM] [homebridge-paradox-securitysystem] This plugin generated a warning from the characteristic 'Security System Target State': SET handler returned write response value, though the characteristic doesn't support write response! See https://git.io/JtMGR for more info.

**Comment by deon-wentzel** [comment]
Hi,

I have been using the latest with no issues. I am running it on the latest version of Hoobs.

**Comment by ElphaX** [comment]
Anyone able to assist me in setting this up?
